x-pack/plugins/apm/public/components/shared/PropertiesTable/__test__/PropertiesTable.test.js

```diff
@@ -56,7 +56,7 @@ describe('PropertiesTable component', () => {
     expect(
       shallow(
         <PropertiesTable
-          propData={{}}
+          propData={null}
```

Comment:
is it save to just omit the `propsData` prop and let the default value be used?

Reply:
We already have a test that checks for this `should render empty when data isn't present`

Reply:
I changed this test to `should still render NestedKeyValueTable even when data has no keys`